src/command_modules/azure-cli-iot/azure/cli/command_modules/iot/custom.py

```diff
@@ -112,6 +138,38 @@ def iot_hub_policy_get(client, hub_name, policy_name, resource_group_name=None):
     return client.get_keys_for_key_name(resource_group_name, hub_name, policy_name)
 
 
+def iot_hub_policy_create(client, hub_name, policy_name, permissions, resource_group_name=None):
+    hub = iot_hub_get(client, hub_name, resource_group_name)
+    policies = iot_hub_policy_list(client, hub_name, hub.resourcegroup)
+    if _is_policy_existed(policies, policy_name):
+        raise CLIError('Policy {0} already existed.'.format(policy_name))
+    updated_policies = []
+    updated_policies.extend(policies)
```

Comment:
This means the `policies` is not a `container`, but rather a `generator`. So the solution is even simpler：
```
upload_policies = [policies]
```